chart/values.yaml

```diff
@@ -515,6 +515,10 @@ mysql:
   initdbScriptsConfigMap: db-init-scripts
   volumePermissions:
     enabled: true
+  resources:
+    requests:
+      cpu: 200m
+      memory: 250Mi
```

Comment:
Nit: I'd vote for:
 - no CPU
 - memory aroung 100Mi
to not clutter core-dev. But let's try this first.